# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/ftp/nsFtpControlConnection.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d netwerk/protocol/ftp/nsFtpControlConnection.cpp
--- a/netwerk/protocol/ftp/nsFtpControlConnection.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/netwerk/protocol/ftp/nsFtpControlConnection.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -100,8 +100,7 @@
   mSocket->SetQoSBits(gFtpHandler->GetControlQoSBits());
 
   // proxy transport events back to current thread
-  if (eventSink)
-    mSocket->SetEventSink(eventSink, GetCurrentThreadEventTarget());
+  if (eventSink) mSocket->SetEventSink(eventSink, GetCurrentEventTarget());
 
   // open buffered, blocking output stream to socket.  so long as commands
   // do not exceed 1024 bytes in length, the writing thread (the main thread)
@@ -134,7 +133,7 @@
   NS_ENSURE_STATE(mSocketInput);
 
   mListener = listener;
-  return mSocketInput->AsyncWait(this, 0, 0, GetCurrentThreadEventTarget());
+  return mSocketInput->AsyncWait(this, 0, 0, GetCurrentEventTarget());
 }
 
 nsresult nsFtpControlConnection::Disconnect(nsresult status) {